apps/frontend/src/pages/Account.svelte

```diff
@@ -1,17 +1,35 @@
-<script>
+<script lang="ts">
   // import { crossfade } from "svelte/types/runtime/transition";
   import Button from '../components/base/Button.svelte';
   import Card from '../components/base/Card.svelte';
   import Dropdown from '../components/base/Dropdown.svelte';
   import Layout from '../components/Layout.svelte';
   import Listbalances from '../components/Listbalances.svelte';
   import SendForm from '../components/SendForm.svelte';
+  import SendModal from '../components/SendModal.svelte';
+  import TxModal from '../components/TxModal.svelte';
+
+  let visible: boolean;
+  const openModal = () => {
+    console.log("hi");
+    visible = true;
+  };
+
+  let txModalVisible: boolean;
+  const openTxModal = () => {
```

Comment:
Idk if you saw this package, but its pretty nice and simple

Reply:
https://www.npmjs.com/package/svelte-simple-modal

Reply:
this is pretty neat, but unfortunately it doesn't take CSS classes so you can't use it with tailwind, so I ended up writing my own version of this (currently without transitions, but those can be added later)